src/main/java/mate/academy/Main.java

```diff
@@ -7,14 +10,33 @@
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
 
-public class Main {
-    public static void main(String[] args) {
-        AuthenticationService authenticationService = new AuthenticationServiceImpl();
+import java.util.logging.Logger;
+
+public final class Main {
+    /**
+     * The logger for the Main class.
+     */
+    private static final Logger LOGGER = Logger.getLogger(
+            Main.class.getName());
+
+    private Main() {
+        // private constructor to prevent instantiation
+    }
+
+    /**
+     * The main method that starts the application.
+     *
+     * @param args the command line arguments
+     */
+    public static void main(final String[] args) {
+        AuthenticationService authenticationService =
+                new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            LOGGER.severe("An authentication exception occurred: "
+                    + e.getMessage());
```

Comment:
You don`t need concatenate string and exception, this method can receive two args error(String, e)
And use method error if you log in catch block